stripProjectRoot by default

jmshal opened the pull request:
The `stripProjectRoot` option is now enabled by default.  I also wrote some tests (using Jest), which uncovered the fact that running the CLI on macOS/Linux for a sourcemap file generated on Windows would fail to produce stripped paths - that has been fixed.

JacobMarshallPP commented:
~Reminder from yourself, @kattrali: please run the tests manually :)~

Edit: I've setup Travis & AppVeyor.

kattrali commented:
😂

jmshal committed:
Enable stripProjectRoot by default (& tests)

- Added Windows path support on macOS/Linux
- Wrote some tests for stripping the project root

jmshal commented on index.test.js:
```diff
@@ -1,7 +1,59 @@
+    )).toBe('/Users/test/git/my-app/src/index.js');
+  });
+
+  test('strips project root from node_moodules', () => {
```
🐮

kattrali commented:
This works perfectly, I think the only thing left is to remove the command-line option for it.

kattrali committed:
Remove unused option flag
